Dockerfile

```diff
@@ -0,0 +1,31 @@
+# https://expressjs.com/en/advanced/best-practice-performance.html
+# https://javascript.plainenglish.io/build-a-production-ready-node-express-api-with-docker-9a45443427a0
+# https://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/
+
+# docker build -t epflsi/startpage .
+# docker run -it epflsi/startpage bash
+# docker run -e PORT=1337 -p 3000:1337 epflsi/startpage -d
+
+FROM node:20-slim
+
+# RUN apt-get -qq update && apt-get -qq -y --no-install-recommends install curl
+# RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
+RUN npm install -g pnpm pm2
```

Comment:
Do we want to version these packages?

Reply:
Maybe later, but right now I can't see why we would like to.